README.md

```diff
@@ -1,6 +1,10 @@
 ## Stream Twitter clone
 
-This is a simple twitter clone app built with [Stream](http://getstream.io)'s API. It shows you how you can use [GetStream.io](https://getstream.io/ "GetStream.io") to built a site similar to Twitter.
+This is a simple Twitter clone app built with [Stream](http://getstream.io)'s API. It shows you how you can use [GetStream.io](https://getstream.io/ "GetStream.io") to built a site similar to Twitter.
+
+You can sign up for a Stream account at https://getstream.io/get_started.
```

Comment:
can make this a hyperlink like `[https://getstream.io/get_started](https://getstream.io/get_started)`?